Source/WebCore/css/CSSGroupingRule.cpp

```diff
@@ -110,10 +111,51 @@ ExceptionOr<void> CSSGroupingRule::deleteRule(unsigned index)
     return { };
 }
 
-void CSSGroupingRule::appendCSSTextForItems(StringBuilder& result) const
+String CSSGroupingRule::cssText() const
 {
-    for (unsigned i = 0, size = length(); i < size; ++i)
-        result.append("  ", item(i)->cssText(), '\n');
+    StringBuilder builder;
+
+    builder.append(" {");
+
+    auto [decls, rules] = cssTextForItems();
```

Comment:
`declarations`

Reply:
`decls` and `rules` are spec vocabulary (cf https://www.w3.org/TR/cssom/#css-rules)